lib/code_corps/model/project.ex

```diff
@@ -42,10 +44,11 @@ defmodule CodeCorps.Project do
     has_many :task_lists, CodeCorps.TaskList
     has_many :tasks, CodeCorps.Task
 
-    has_many :categories, through: [:project_categories, :category]
-    has_many :skills, through: [:project_skills, :skill]
     has_many :stripe_connect_subscriptions, through: [:stripe_connect_plan, :stripe_connect_subscriptions]
 
+    many_to_many :categories, CodeCorps.Category, join_through: CodeCorps.ProjectCategory
+    many_to_many :skills, CodeCorps.Skill, join_through: CodeCorps.ProjectSkill
```

Comment:
Probably wanna do an `on_delete: :delete_all` here. Default is `:nothing` according to the docs.

If this works out, I wouldn't mind doing it with tasks and users where applicable to.